corehq/apps/events/models.py

```diff
@@ -18,6 +19,9 @@
     (ACCEPTED, _('Accepted')),
 ]
 
+# Attendees are CommCareCase instances with this case type:
+ATTENDEE_CASE_TYPE = 'commcare-attendee'
```

Comment:
Good point. I think we should reserve this.

Reply:
The prefix "commcare-" was chosen so that app builders are unlikely to _accidentally_ name a case type with the same name. But they should be able to do it _intentionally_.

The reason we didn't make this reserved (the way we do for "commcare-user") is a _allow_ Event Managers to import attendees. (You can't import user cases in HQ.) The spec says: 

> Web users can use Case Import to import attendee cases. They must have “commcare-attendee” as their case type.

We also want app builders to be able to build and customize apps that create and manage attendee cases. The spec goes on,

> The Attendees page uses CRUDPaginatedView to list cases of case type “commcare-attendee” by name. It offers the ability to add an attendee case. The form only prompts for the “name” case property.

The reason we only assume the existence of the "name" property is because we want to be flexible enough to allow projects to decide the schema of their own attendees. Are they mobile workers? Are they farmers? Students? Kids at a day care? We should not be nailing that down. Each project should be able to build attendance tracking around their own use case.